tests/test_core.py

```diff
@@ -7271,6 +7271,7 @@ def test_embind_unsigned(self):
     self.emcc_args += ['--bind']
     self.do_run_from_file(path_from_root('tests', 'embind', 'test_unsigned.cpp'), path_from_root('tests', 'embind', 'test_unsigned.out'))
 
+  @no_asan('FIXME #11158')
```

Comment:
i guess this can go now that the fixed embind is in.

Reply:
Sadly no, the test still fails, so there are more unsafety issues in the embind JS code,

https://github.com/emscripten-core/emscripten/issues/11158

It's possible none of them are actually dangerous, but we should investigate.